plugin/rewrite/name.go

```diff
@@ -1,29 +1,116 @@
 package rewrite
 
 import (
-	"github.com/coredns/coredns/plugin"
-
-	"github.com/miekg/dns"
+    "github.com/coredns/coredns/plugin"
+    "github.com/miekg/dns"
+    "fmt"
+    "strings"
+    "regexp"
+    "strconv"
 )
```

Comment:
What are these changes? Is this a different `go fmt` or something?

Reply:
I use vim-go. Will format with “go fmt” next time I rebase.